lib/load-simple-icons.js

```diff
@@ -14,7 +14,8 @@ function loadSimpleIcons() {
   // https://github.com/badges/shields/issues/4273
   Object.keys(originalSimpleIcons).forEach(key => {
     const icon = originalSimpleIcons[key]
-    const title = icon.title.toLowerCase()
+    const title = icon.title
```

Comment:
Entire diff is based on a different key structure in v7. Think this is best illustrated with a concrete example.

In v6 & v5, the imported object's keys included `hive` and `hive_blockchain` for their respective icons, whereas in v7 the same corresponding keys are `siHive` and `siHive_blockchain`. This in turn requires some juggling in how we're making comparisons